server/spec/factories/experiences.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+
+  factory :experience do
+    available_offline {Faker::Boolean.boolean(true_ratio: 1)}
```

Comment:
Is this meant to be "on all the time"?

If so, then the value in the block can be true, instead of asking for it via Faker.

Reply:
I made it true all the time because model validation will error if both 'available_offline' and 'available_online' are false. I have set it to true all the time so no invalid users will be created by the factory.